orttraining/orttraining/core/optimizer/memory_optimizer/memory_optimizer.cc

```diff
@@ -312,6 +321,12 @@ Status MemoryOptimizer::CreateRecomputeGraph(Graph& graph,
 
       graph.AddConsumerNode(input_arg->Name(), &recompute_node);
     }
+
+    bool training_mode_reset = SetTrainingModeForForwardPythonOpNode(*node_to_duplicate);
```

Comment:
nit: why not output log during the function body of SetTrainingModeForForwardPythonOpNode, thus the function not needs to return a flag

Reply:
Oh, did not think too much for it. I think it is okay because in that way, we still need add a logger pram for SetTrainingModeForForwardPythonOpNode also.